src/plugins/wb-disable/disable.js

```diff
@@ -30,15 +30,29 @@ var componentName = "wb-disable",
 			$html = wb.html,
 			i18n = wb.i18n,
 			pageUrl = wb.pageUrlParts,
-			li, param;
+			li, param,
+			i18nTemporary = {
```

Comment:
Yeah their a google spreadsheet. I don't think it documented with the public.

https://docs.google.com/spreadsheets/d/1BmMrKN6Rtx-dwgPNEZD6AIAQdI4nNlyVVVCml0U594o/edit#gid=1